src/lib/Scenes/Artwork/Components/ArtworkExtraLinks.tsx

```diff
@@ -61,10 +63,10 @@ export class ArtworkExtraLinks extends React.Component<ArtworkExtraLinksProps> {
   renderFAQAndSpecialist = () => {
     const {
       artwork: { isAcquireable, isOfferable, isInAuction, sale, isForSale },
+      auctionState,
     } = this.props
 
-    if (isInAuction && sale && !sale.isClosed && isForSale) {
-      // FIXME: Verify logic when to show Auction "ask a specialist"
+    if (isInAuction && sale && !sale.isClosed && isForSale && auctionState !== "hasEnded") {
```

Comment:
I think we can remove the check for `sale.isClosed` here (since `auctionState` should handle that)